core/src/main/scala/org/bitcoins/core/protocol/script/ScriptWitness.scala

```diff
@@ -94,6 +99,13 @@ sealed abstract class P2WSHWitnessV0 extends ScriptWitnessV0 {
     RawScriptPubKey.fromBytes(cmpct.bytes ++ stack.head)
   }
 
+  lazy val pubKeys: Seq[ECPublicKey] = {
+    val pubKeys: Seq[ScriptToken] = redeemScript.asm
+      .filter(_.isInstanceOf[ScriptConstant])
+      .filterNot(_.isInstanceOf[ScriptNumberOperation])
+    pubKeys.map(k => ECPublicKey(k.hex)).filter(_.isFullyValid)
```

Comment:
Why the `.filter(_.isFullyValid)` here? Is it to try and get rid of other things that may be a `ScriptConstant`, but not a `ECPublicKey`?

Reply:
Yes

Reply:
yup